examples/Huggingface_Transformers/Transformer_handler_generalized.py

```diff
@@ -82,6 +81,23 @@ def initialize(self, ctx):
                 self.model = AutoModelForCausalLM.from_pretrained(model_dir)
             else:
                 logger.warning("Missing the operation mode.")
+            #Using the Better Transformer integration to speedup the inference
+            if self.setup_config["BetterTransformer"]:
+                try:
+                    from optimum.bettertransformer import BetterTransformer
+                    is_optimum_available = True
+                except ImportError as error:
+                logger.warning(
+                    "HuggingFace Optimum is not installed. Proceeding without BetterTransformer"
+                )
+                if is_optimum_available:
```

Comment:
This works but I think it's better to avoid deep nesting like this, it's more common to do 

```
try: 

except Error1:

except Error2:
```

or 

```
try:

except Error1, Error2:
```